ReactWindows/ReactNative.Net46/Views/Text/ReactTextShadowNode.cs

```diff
@@ -238,6 +275,7 @@ private void UpdateTextBlockCore(TextBlock textBlock, bool measureOnly)
             textBlock.FontSize = _fontSize ?? 15;
             textBlock.FontStyle = _fontStyle ?? new FontStyle();
             textBlock.FontWeight = _fontWeight ?? FontWeights.Normal;
+            textBlock.TextDecorations = new TextDecorationCollection(TextDecorationCollection);
 
```

Comment:
I can be wrong, but from what I see, <Text> node is the lowest-level possible for creating styled text components. I cannot see a way to apply a style to a raw text. It would be great if you can come up with any example to understand the problem.